.storybook/icon-component-loader.js

```diff
@@ -0,0 +1,8 @@
+function IconComponentLoader(source) {
+  return source.replace(
+    `import icons from '../../icons/spritemap.svg';`,
+    `const icons = "spritemap.svg";`,
+  );
+}
+
+module.exports = IconComponentLoader; // eslint-disable-line no-undef
```

Comment:
Not sure what this file does (brought over from `next` and is supposedly used in the storybook webpack config), but eslint complains about `module` use.

Reply:
![image](https://user-images.githubusercontent.com/86632227/164049779-0d836960-68a0-4b33-b9c3-1b07179ffbc2.png)

As for what it does, I'll try removing and see if any errors occur

Reply:
Removed and storybook seems to build and publish to Chromatic fine: https://www.chromatic.com/build?appId=61313967cde49b003ae2a860&number=1130, think I'll remove this and if errors can come back to it